_posts/2015-08-09-Stats-Reset.markdown

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title:  "CocoaPods Stats Reseting"
+author: orta
+categories: cocoapods website
+---
+
+TL;DR: _CocoaPods Stats is ready to move from it's current staging environment_, and we will be resetting all stats next weekend.
+
+<!-- more -->
+
+When I set up CocoaPods Stats, the API + analytics database with the help from [Segment](http://segment.com). I created it within a staging environment in our analytics database. I gave it two months to gather enough data in order for us to be sure that the at-the-time evolving table data structure works. 
```

Comment:
Fixed in  https://github.com/CocoaPods/blog.cocoapods.org/commit/c7df06a2d2c774159b456ce5a447f3457854fb97